docs/licenses.rst

```diff
@@ -0,0 +1,63 @@
+.. _license_types:
+
+License types
+=============
+
+To log in to Signavio Workflow, your user account have a user license that has not expired.
+You can choose between several license types:
+
+* Enterprise Cloud Free Trial
+* Enterprise Cloud
+* Professional Cloud
+* Developer Cloud
+* Academic Cloud
+
+The *Feature Overview* section on the `Signavio Workflow product page <http://www.signavio.com/products/workflow/>`_ shows the main differences between these versions.
+These differences affect:
+
+* the license expiry period - typically one or twelve months
+* which software features you can use
+* the amount of user file storage
+* the support level.
+
+The :ref:`billing` page in the application shows details of the current license type.
+
+
+Enterprise Cloud Free Trial
+---------------------------
+
+The free trial license lets you try the full version of the software for a limited period without purchasing a license.
+To create a trial license, use the `registration <https://workflow.signavio.com/registration>`_ page.
+When the trial expires, Signavio Workflow will send you an email inviting you to purchase licenses.
+
+
+Enterprise Cloud
+----------------
+
+A *Enterprise Cloud* license gives you full access to all features with minimum limitations.
+To purchase *Enterprise Cloud* licenses, contact `sales@signavio.com <mailto:sales@signavio.com?subject=Signavio%20Workflow%20Enterprise%20Cloud>`_.
+
+
+Professional Cloud
+------------------
+
+A *Professional Cloud* license gives you access to core features for a lower license cost than *Enterprise Cloud*.
+To purchase *Professional Cloud* licenses, contact `sales@signavio.com <mailto:sales@signavio.com?subject=Signavio%20Workflow%20Professional%20Cloud>`_.
+
+
+Developer Cloud
+---------------
+
+A free *Developer Cloud* license allows developers to create and test process models without having to purchase a license.
+You can only use this license in an organisation with a single user, which means that multiple people cannot use it for collaborative process execution.
+To downgrade a license to a *Developer Cloud* license, use the `purchasing <https://workflow.signavio.com/buy>`_ page.
+
+
+Academic Cloud
+--------------
+
+A free *Academic Cloud* license allows university students to use Signavio Workflow to learn about workflow management.
+You may only use this license if you have a valid university email address.
+You may only use this license for educational purposes and not for administrative processes in an educational institution.
+To register for an academic license, use the *academic version* link at the bottom of the `purchasing <https://workflow.signavio.com/buy>`_ page.
```

Comment:
Should you mention that for now they have to register for a normal trial and can upgrade to the academic version afterwards?